src/screens/TypeScreen.tsx

```diff
@@ -43,7 +43,8 @@ export const TypeScreen: React.FC<TypeScreenProps> = ({ navigation }) => {
   const [account, setAccount] = useState(user.name || '')
 
   const handleAddSecretButtonPress = async () => {
-    await add({ uid: user.uid, secret, account, issuer })
+    const cleanedSecret = secret.trim().replace(/ /g, '').toUpperCase()
```

Comment:
what if you remove the trim function and use `/\s/g` as replace regex?

Reply:
Yeah, you can remove `trim` in favor to use only `replace` with regex suggested by @ilteoood .

Reply:
Will do. I've blindly copy-pasted 😅